applications/common/backend/src/main/java/pl/ds/kyanite/common/components/services/impl/ContactFormConfigStoreImpl.java

```diff
@@ -16,46 +16,31 @@
 
 package pl.ds.kyanite.common.components.services.impl;
 
-import java.util.ArrayList;
-import java.util.List;
 import org.osgi.service.component.annotations.Component;
 import org.osgi.service.component.annotations.Reference;
 import org.osgi.service.component.annotations.ReferenceCardinality;
 import org.osgi.service.component.annotations.ReferencePolicy;
 import pl.ds.kyanite.common.components.services.ContactFormConfigStore;
 import pl.ds.kyanite.common.components.services.ContactFormConfiguration;
+import pl.ds.kyanite.common.components.services.config.BaseSpaceDependentConfigStore;
 
-@Component(service = ContactFormConfigStore.class, immediate = true)
-public class ContactFormConfigStoreImpl implements ContactFormConfigStore {
 
-  private List<ContactFormConfiguration> configList = new ArrayList<>();
+@Component(service = ContactFormConfigStore.class, immediate = true)
+public class ContactFormConfigStoreImpl
+    extends     BaseSpaceDependentConfigStore<ContactFormConfiguration>
```

Comment:
Do you use a https://github.com/google/styleguide/blob/gh-pages/intellij-java-google-style.xml? 
It looks clear, but I'm concern that the first person that would do some changes and use a shortcut to reformat the code will have changes here.

Reply:
You're right. I've fixed the indents but kept the line breaks.
I use Google style schema but I don't use the shortcut that much. Stylechecker didn't complain so I didn't think it would be an issue.